tests/admin_scripts/tests.py

```diff
@@ -1822,6 +1844,50 @@ def test_label_command_multiple_label(self):
             "('settings', None), ('traceback', False), ('verbosity', 1)]"
         )
 
+    def test_with_suppress_command_help(self):
+
+        args = ['with_suppress_command', '--help']
+        out, err = self.run_manage(args)
+        self.assertNoOutput(err)
+        self.assertOutput(out, 'Test basic commands with suppress')
+        self.assertOutput(out, '--help')
+        self.assertNotInOutput(out, '--verbosity')
+        self.assertNotInOutput(out, '--traceback')
+        self.assertNotInOutput(out, '--pythonpath')
+        self.assertNotInOutput(out, '--settings')
+        self.assertNotInOutput(out, '--version')
+        self.assertNotInOutput(out, '--no-color')
+        self.assertNotInOutput(out, '--force-color')
```

Comment:
Please use the same order as in `--help` output, i.e. `--version`, `--verbosity`, `--settings`, `--pythonpath`, `--traceback`, `--no-color`, and `--force-color`.